gpu/src/Texture.zig

```diff
@@ -211,9 +211,9 @@ pub const SampleType = enum(u32) {
 
 pub const BindingLayout = extern struct {
     reserved: ?*anyopaque = null,
-    sample_type: SampleType,
-    view_dimension: TextureView.Dimension,
-    multisampled: bool,
+    sample_type: SampleType = .none,
+    view_dimension: TextureView.Dimension = .dimension_none,
+    multisampled: bool = false,
```

Comment:
`.float` and `.dimension_2d`

<img width="493" alt="image" src="https://user-images.githubusercontent.com/3173176/162637574-750b8c1a-62f5-42a0-9b07-5e7f20f29c35.png">